packages/reports/src/Constants/data-table-constants.tsx

```diff
@@ -25,20 +37,43 @@ const getModeFromValue = key => {
         transfer: 'transfer',
     };
 
-    if (Object.keys(map).find(x => x === key)) {
-        return map[key];
-    }
+    return map[key] || map.default;
+};
+
+type TCellContentProps = {
+    cell_value: string;
+    passthrough: any;
+    row_obj: any;
+    is_footer: boolean;
+};
+
+type THeaderProps = {
+    title: React.ReactNode;
+};
+
+type TColumnTemplateType = {
+    key?: string;
+    title?: React.ReactNode;
+    col_index?: string;
+    renderCellContent?: (props: TCellContentProps) => React.ReactNode;
+    renderHeader?: (props: THeaderProps) => React.ReactNode;
+};
 
-    return map.default;
+type TMultiplierOpenPositionstemplateProps = {
+    currency: string;
+    onClickCancel: () => void;
+    onClickSell: () => void;
+    getPositionById: (id: string) => any;
+    server_time: object;
```

Comment:
It's better to use generic object type instead of objet

Reply:
Thanks, changed👍🏻